iOS Example Frame/iOS Example Frame/CustomStyle2.swift

```diff
@@ -41,10 +42,7 @@ struct PaymentHeaderCellStyleCustom2: PaymentHeaderCellStyle {
   var backgroundColor = Constants.greenBackgroundColor
   var headerLabel: ElementStyle? = HeaderLabelFormStyleCustom2(text: "Payment details")
   public var subtitleLabel: ElementStyle? = TitleLabelStyleCustom2(text: "Visa, Mastercard and American Express accepted.")
-  public var schemeIcons: [UIImage?]? = [
-    UIImage(named: "visa-icon"),
-    UIImage(named: "mastercard-icon"),
-    UIImage(named: "amex-icon")]
+  public var schemeIcons: [UIImage?]? = nil
```

Comment:
Do we have some task somewhere to fix this optional array of optional images? We really should be able to reduce at least one of the optionals, if not even both 🤔

Reply:
There're 2 cases: 
1- The merchant may not want to show any images then  they can set it to `= nil`
2- `UIImage` is optional by default. they may send an image which was removed by mistake from their side then we don't show it.

Reply:
1. Wouldn't they just provide 0 images (ie: `[]`) ? Swift encourages removing optionality where possible so I would think that an empty array will always be easier to deal with